wiki.js

```diff
@@ -0,0 +1,56 @@
+//Module for getting 'clues' from wikipedia, EG flags or language
+
+// test stuff
+var country = {country: 'Germany'};
+
+
+
+/**
+* Uses the object from the flickr request to get information from wikipedia to
+* generate a request for the wikipedia information. Returns a promise.
+*/
+function getCountryClues(flickrCountry) {
+  var PARAMS = {
+    'action': 'query', 'titles': flickrCountry['country'], format: 'json',
```

Comment:
…but in using the photo object, you can just use dot notation: `flickrCountry.country`.

I would be remiss to mention that if the variable name seems redundant, it's probably the wrong one to use.